src/main/java/com/programmers/ticketparis/member/controller/CustomerController.java

```diff
@@ -30,6 +31,16 @@ public CustomerIdResponse createCustomer(@Valid @RequestBody CustomerCreateReque
         return customerService.createCustomer(customerCreateRequest);
     }
 
+    @PostMapping("/login")
+    @ResponseStatus(HttpStatus.CREATED)
+    public CustomerResponse login(@Valid @RequestBody CustomerLoginForm customerLoginForm) {
+        CustomerResponse loginCustomerResponse = customerService.login(customerLoginForm);
```

Comment:
`p4`

주 도메인이 앞에 나오는 것이 조금 더 명확한 것 같은데 어떻게 생각하시나요?!

```suggestion
        CustomerResponse customerLoginResponse = customerService.login(customerLoginForm);
```

Reply:
주 도메인이 앞에 오는 것이 명확한 것 같네요!! 수정했습니다~!
957837000fc1022081a7a1613d11c8f2da0bc89e